presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSourceFactory.java

```diff
@@ -206,6 +207,9 @@ public static OrcPageSource createOrcPageSource(
             if (e.getClass().getSimpleName().equals("BlockMissingException")) {
                 throw new PrestoException(HIVE_MISSING_DATA, message, e);
             }
+            else if (e.getClass().getSimpleName().equals("InvalidProtocolBufferException")) {
```

Comment:
Do you have a stack trace for this one?  I would expect the ORC reader to convert this to a Presto exception